src/shared/web-components/src/FamilyHubs.SharedKernel.Razor/Cookies/ICookiePage.cs

```diff
@@ -6,6 +6,14 @@ public interface ICookiePage
 {
     string CookiePolicyContent { get; }
     bool ShowSuccessBanner { get; }
+    ConsentCookie? Cookie { get; }
 
+    void OnGet(HttpRequest request);
     void OnPost(bool analytics, HttpRequest request, HttpResponse response);
-}
\ No newline at end of file
+
+    public record ConsentCookie
```

Comment:
Can this not be written as:

public record ConsentCookie(bool analytics, int version);

Result in the same thing?

Reply:
Fair enough. Forgot about where it is used.